src/components/guide/index.js

```diff
@@ -0,0 +1,221 @@
+              </a>
+            </li>
+          ))}
+        </ul>
+      );
+    }
+  }
+  setGuide() {
+    const params = {assessmentId: this.props.assessmentID, localeKey: this.props.locale};
+    this.props.traitify.graphqlQuery("/interview_guides/graphql", guideQuery({params}))
+      .then((response) => {
+        if(response.errors) {
+          this.processError(response.errors);
+          this.setState({errors: response.errors});
+          return;
+        }
+        const {competencies} = response.data.guide;
+        const badges = this.badges(competencies);
+        this.setState({badges, competencies, displayedCompetency: competencies[0]});
+      });
+  }
+  badges(competencies) {
+    const {assessment} = this.props;
+
+    return competencies.map((competency) => {
+      const personalityId = competency.questionSequences[0].personality_type_id;
+      const types = assessment.personality_types;
+      const personality = types.filter((type) => type.personality_type.id === personalityId)[0];
+      return {image: personality.personality_type.badge.image_medium, name: competency.name};
+    });
+  }
+  stringToListItems(entity) {
+    let entities = entity.split("\n");
+    entities = entities.map((e) => <li key={listId()}>{e}</li>);
+
+    return (
+      <ul>
+        {entities}
+      </ul>
+    );
+  }
+  tabBadge(tab) {
+    return this.state.badges.filter((badge) => badge.name === tab)[0].image;
+  }
+  render() {
+    if(this.state.errors.length > 0) { return <div />; }
+    if(this.state.competencies.length === 0) { return <div />; }
+    if(!this.props.assessment) { return <div />; }
+
+    const {displayedCompetency} = this.state;
+    const {translate} = this.props;
+    const {intro, readMore} = this.introduction();
+
+    return (
+      <div className={style.tabsContainer}>
+        <div className={style.tabContainer}>
+          {this.selectBoxOrTabs()}
+        </div>
+
+        <div className={style.tabsContent}>
+          <div className={style.tabContentActive}>
+            <h2>{displayedCompetency.name}</h2>
+            {intro}
+            <p>
+              <a
+                href="#read-more"
+                tabIndex={0}
+                onClick={() => this.handleReadMore()}
+                onKeyPress={() => this.handleReadMore()}
+              >
+                {translate("read_more")}
+              </a>
+            </p>
+            {this.expandedIntro(readMore)}
+            <hr />
+            {displayedCompetency.questionSequences.map((sequence) => (
+              <div key={sequence.id}>
+                <h2>{sequence.name}</h2>
+                <p>{translate("guide_intro")}</p>
+                <p><em {...dangerousProps({html: translate("guide_get_started_html")})} /></p>
+                {sequence.questions.map((question) => (
+                  <div key={question.id}>
+                    <h3 id={question.order === 1 ? "question-1" : null}>{`Question ${question.order}`}</h3>
```

Comment:
I think we might want something more unique for the ID. In case it's embedded on a page with similar IDs. Something like traitify-question-1 might work.

I don't really understand why we have it at all since it just brings you to the next section...but I guess we just have to follow the design